tests/test_controller.py

```diff
@@ -65,7 +65,7 @@ def verify_call(aioresponse: tuple, method: str, url: str, **kwargs: dict) -> bo
             successful_match = True
 
             for key in kwargs:
-                if key not in call[1] and call[1][key] != kwargs[key]:
+                if key not in call[1] or call[1][key] != kwargs[key]:
```

Comment:
Interesting, well if all tests continue to pass.

Reply:
I ran the existing tests that used that function and they do pass, but I had to fix some mistakes in the tests I added.

Reply:
I ran the rest of the tests and found two that fail after this change in test_controller.py.  test_controller() fails at line 106.
`call[1][key]: {'username': 'user', 'password': 'pass', 'remember': True}`
`kwargs[key]: {'username': 'user', 'password': 'password', 'remember': True}`

test_unifios_controller fails at line 199 and 212.
`call[1][key]: {}`
`kwargs[key]: {'x-csrf-token': '123'}`

`call[1][key]: {'username': 'user', 'password': 'pass', 'remember': True}`
`kwargs[key]: {'username': 'user', 'password': 'password', 'remember': True}`

I have not committed fixes for these tests, since I didn't write them.  Should I revert the change to verify_call() since it's not strictly required to add support for outlets and is probably out of scope for this PR?

Reply:
Yeah change it back, lets manage that separately